ArchUnitNET/Fluent/Syntax/Elements/Members/PropertyMembers/IPropertyMembersShould.cs

```diff
@@ -0,0 +1,103 @@
+﻿using ArchUnitNET.Domain;
+
+namespace ArchUnitNET.Fluent.Syntax.Elements.Members.PropertyMembers
+{
+    public interface IPropertyMembersShould : IMembersShould<PropertyMembersShouldConjunction, PropertyMember>
+    {
+        /// <summary>
```

Comment:
please either add documentation to all methods or to none

Reply:
I added the documentation for these methods because it isn't clear what the inherited conditions BePublic(),BePrivate(),... mean for properties. I will add documentation for all methods in the future.